x/mongo/driver/topology/connection.go

```diff
@@ -30,6 +30,8 @@ import (
 
 var globalConnectionID uint64 = 1
 
+var maxMessageSizeDefault uint32 = 48000000
```

Comment:
nit: Rename to `defaultMaxMessageSize`. It's a bit more consistent with some of our existing constants.

Reply:
Sounds good!